libgeometry/src/geo_surface.c

```diff
@@ -196,6 +196,29 @@ void geo_surface_fprintf_irap_external_zcoord( const geo_surface_type * surface,
   geo_surface_fprintf_irap__( surface , filename , zcoord );
 }
 
+geo_surface_type  * geo_surface_alloc_new( int ny, double xinc, double yinc,
```

Comment:
The order of arguments here was quite funny:

1. `x` is *always* before `y`.
2. Either all `x` arguments and then all `y` arguments as: `nx, xstart, xinc, ny, ystart,yinc` or alternating: `nx,ny,xstart,ystart,xinc,yinc`